toolkit/tools/pkg/imagecustomizerlib/imagecustomizer.go

```diff
@@ -679,6 +679,13 @@ func customizeImageHelper(buildDir string, baseConfigPath string, config *imagec
 	}
 	defer imageConnection.Close()
 
+	imageConnection.Chroot().UnsafeRun(func() error {
+		distro, version := getDistroAndVersion()
+		logger.Log.Infof("Base OS distro: %s", distro)
```

Comment:
nit - should it be `Host OS` instead?

Reply:
These logs are for the base OS. Though I assume this comment is for the other file.